run-checks

```diff
@@ -55,6 +55,12 @@ case "${1:-all}" in
     --spread)
         SPREAD=1
```

Comment:
Would it make sense to have `SPREAD=full` here?

Reply:
I agree with mvo. There's no reason to use integers for this.